### PR TITLE
Renaming to TestApps/TestAppSwiftUI

### PR DESCRIPTION
**Pull request by emdobrin**
## Description

- Renamed test apps folder to TestApps
- Renamed test app to TestAppSwiftUI
- Updated CircleCI Xcode version to 12.0.1 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by cacheung — APPROVED** [review]
Able to run TestApp_SwiftUI.

**Review by kevinlind — APPROVED** [review]
Everything looks good, however I would prefer the name `TestAppSwiftUI`, without the underscore. Just a personal preference.

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
 
   build-and-test:
     macos:
-      xcode: 11.6.0 # Specify the Xcode version to use
+      xcode: 12.0.1 # Specify the Xcode version to use
```
Comment (kevinlind) [inline .circleci/config.yml:17]: You'll need to update the iOS simulator version as well. It looks like "iPhone 8 (13.5)" is a valid simulator in the 12.0.1 image.
Here's the AEPAudience CircleCI build which lists the simulators for the 12.0.1 image:
https://app.circleci.com/pipelines/github/adobe/aepsdk-audience-ios/218/workflows/23e1af05-5c85-4455-97ab-845dbdefc192/jobs/225?invite=true#step-111-74

**Review by kevinlind — APPROVED** [review]
Looks good. I added a comment about updating the CircleCI configuration with a valid simulator version.

**Pull request merged**